src/components/ComputeModal.js

```diff
@@ -176,7 +176,6 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
   const [customEnvImage, setCustomEnvImage] = useState('')
   const [jupyterUserScriptUri, setJupyterUserScriptUri] = useState('')
   const [sparkMode, setSparkMode] = useState(false)
-  const [bucketLocation, setBucketLocation] = useState(defaultLocation)
```

Comment:
Thanks, the changes in this file look right to me. We have the computeRegion and computeZone states holding the location of the to-be created cloud env, and we have the location prop to hold the location of the workspace bucket. Thus, there is no need for the bucketLocation state here.